cmake/modules/python.cmake

```diff
@@ -31,10 +31,9 @@ if(NOT Python3_EXECUTABLE)
                                  "import sys; sys.stdout.write('.'.join([str(x) for x in sys.version_info[:2]]))"
                          RESULT_VARIABLE result
                          OUTPUT_VARIABLE version
-                         ERROR_QUIET
                          OUTPUT_STRIP_TRAILING_WHITESPACE)
 
-       if(version VERSION_LESS PYTHON_MINIMUM_REQUIRED)
+       if(NOT version OR version VERSION_LESS PYTHON_MINIMUM_REQUIRED)
```

Comment:
The `NOT version` is not needed.
Undefined version / empty stings is treated as version zero, '0'.

And that is stated in the docs.
> omitted components are treated as zero

`major` is missing in undefined / empty string, and this major is `0`.

Reply:
Done, thanks. I think it was also off-topic....